### PR TITLE
feat(icons): Add CollectionItemLink icon

### PR DESCRIPTION
**Pull request by shahzadaziz**
Screen Shot 2019-08-16 at 1 37 33 PM

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @shahzadaziz has signed the CLA. Thanks for the pull request!

File: src/icons/two-toned/IconCollectionItemLink.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+const IconCollectionItemLink = ({ className = '', height = 24, title, width = 24 }: TwoTonedIcon) => (
+    <AccessibleSVG
+        className={`bdl-IconCollectionItemLink ${className}`}
+        height={height}
+        title={title}
+        viewBox="0 0 32 32"
+        width={width}
+    >
+        <g fill="none" fillRule="evenodd">
```
Comment (priyajeet) [inline src/icons/two-toned/IconCollectionItemLink.js]: <g> wrapper likely unnecessary

File: src/icons/two-toned/IconCollectionItemLink.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+import type { TwoTonedIcon } from '../flowTypes';
+import { bdlGray, white } from '../../styles/variables';
+
+const IconCollectionItemLink = ({ className = '', height = 24, title, width = 24 }: TwoTonedIcon) => (
```
Comment (priyajeet) [inline src/icons/two-toned/IconCollectionItemLink.js]: I would start defaulting new icons to 32 32 also, @booc0mtaco ?

Comment (j3tan) [reply src/icons/two-toned/IconCollectionItemLink.js]: most of the two-toned default to 24 right now

Comment (j3tan) [reply src/icons/two-toned/IconCollectionItemLink.js]: but yeah defaulting to 32px seems reasonable seeing as that's the viewbox

**Comment by shahzadaziz** [comment]
@priyajeet addressed your comments.

File: src/icons/two-toned/IconCollectionItemLink.js (hunk 2 above)
Comment (booc0mtaco) [reply src/icons/two-toned/IconCollectionItemLink.js]: i think using default values equal to the viewbox could work. getting into the habit of specifying the width/height where the icons are used would make future refactoring easier, as well

**Comment by booc0mtaco** [comment]
let's get this icons such that it doesn't have embedded padding. that allows us to specify a `border-radius` circle around it, if needed

**Comment by shahzadaziz** [comment]
@booc0mtaco removed icon padding.

**Pull request merged**